samples/benchmarks/coremark/boards/nrf54h20dk_nrf54h20_cpurad.overlay

```diff
@@ -40,3 +40,18 @@
 	owned-channels = <6>;
 	status = "okay";
 };
+
+/* Disable the default UART node for the radio core, as logs are forwarded
+ * with STM and the application core UART. This setting also prevents access
+ * issues to the same UART instance from two or more different cores.
+ */
+&uart135 {
+	status = "disabled";
+};
+
+/* DTS nodes required for the STM standalone logging, imported from the nordic-log-stm snippet. */
+&tddconf {
```

Comment:
Is there  similar node needed for ppr core? Or is it only required by arm cores?

Reply:
The app core configures this DTS node for Pepper as part of the application core DT. The app core owns Pepper